references.bib

```diff
@@ -1,3 +1,12 @@
+@inproceedings{Waheed2022a,
+	author = {Asim Waheed and Sara Qunaibi and Diogo Barradas and Zachary Weinberg},
+	title = {Darwin's Theory of Censorship: Analysing the Evolution of Censored Topics with Dynamic Topic Models},
+	year = {2022},
+	publisher = {Association for Computing Machinery},
+	url = {https://doi.org/10.1145/3559613.3563206},
```

Comment:
Let's change this to a URL that's publicly accessible.
```suggestion
	url = {https://research.owlfolio.org/pubs/2022-darwin-censorship.pdf},
```

Reply:
As the controller of `research.owlfolio.org` I confirm that that is intended to be a permanently stable URL.

Can we keep the DOI value around though, please? As a separate field, `doi = {10.1145/3559613.3563206}` maybe?

Reply:
I haven't been doing this for other records but I can make an exception here.